src/wasm/nxt_wasm.h

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) Andrew Clayton
+ * Copyright (C) F5, Inc.
+ */
+
+#ifndef _NXT_WASM_H_INCLUDED_
+#define _NXT_WASM_H_INCLUDED_
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include <nxt_unit.h>
+
+#include <wasm.h>
+#if defined(NXT_HAVE_WASM_WASMTIME)
+#include <wasmtime.h>
+#endif
+
+#define NXT_WASM_PAGE_SIZE          (64 * 1024)
+
+#define NXT_WASM_MEM_PAGES          512UL       /* 32 MiB */
+
+#define NXT_WASM_MEM_SIZE           (NXT_WASM_PAGE_SIZE * NXT_WASM_MEM_PAGES)
```

Comment:
This will allow removing the comment:

```c
#define NXT_WASM_PAGE_SIZE  (64 * 1024)
#define NXT_WASM_MEM_SIZE   (32UL * 1024 * 1024)
#define NXT_WASM_MEM_PAGES  (NXT_WASM_MEM_SIZE / NXT_WASM_PAGE_SIZE)
```